src/settings.rs

```diff
@@ -10,6 +10,7 @@ use std::path::{Path, PathBuf};
 #[derive(Deserialize, Debug, Clone)]
 pub struct Policy {
     pub url: String,
+    pub mutating: bool,
```

Comment:
I think we should name it in a different way. The policy server doesn't need to know if every policy is mutating or not. It can be made aware of **certain** policies that have to be "downgraded" from mutating to validating.

I would change this attribute to be named something like `downgrade_to_validation`. This would be `false` by default, which is good -> not specifying a value for that means "let the policy do what the author intended".

Reply:
I would be more explicit if we are not going with `mutate: bool`. I would say `allowed_to_mutate: bool` instead.

Reply:
naming is always tough :)

I like `allowed_to_mutate` too